lib/babe/verify_test.go

```diff
@@ -645,21 +640,21 @@ func Test_verifier_verifyAuthorshipRight(t *testing.T) {
 			name:     "valid digest items, getAuthorityIndex error",
 			verifier: *babeVerifier5,
 			header:   header7,
-			expErr:   errors.New("failed to get authority index no digest provided"),
+			expErr:   errNoDigest,
 		},
 		{
 			name:     "get header err",
 			verifier: *babeVerifier6,
 			header:   header7,
-			expErr:   errors.New("failed get header get header error"),
+			expErr:   errors.New("failed to get header for block"),
 		},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			b := &tt.verifier
 			err := b.verifyAuthorshipRight(tt.header)
 			if tt.expErr != nil {
-				assert.EqualError(t, err, tt.expErr.Error())
+				assert.ErrorContains(t, err, tt.expErr.Error())
```

Comment:
Revert this and change the expected errors to `checking for justification: test mock error` and `getting justification: test mock error` for their respective test case.

Reply:
There are hashes included in the some error context, for example 
```
could not verify block equivocation for header 0x2eb61ed30b3663f18c53f29d0d6ad3dd7f40aafdbc73b04727e5e3e9891d9a8c: failed to get authority index for block 0x0100000000000000000000000000000000000000000000000000000000000000: for block hash 0xdcdd89927d8a348e00257e1ecc8617f45edb5118efff3ea2f9961b2ad9b7690a: no digest provided
```

Can't find a test case where `checking for justification: test mock error` and `getting justification: test mock error` are expected test cases

Reply:
addressed